### PR TITLE
Wind model simplified

### PR DESCRIPTION
**Pull request by skb2014**
Improve JSON storage and usage.

### CODE REVIEW

File: Wind-Model/my-app/src/index.js [hunk 1]
```diff
@@ -6,23 +6,7 @@ import reportWebVitals from './reportWebVitals';
...
-import justwindy from './just_windy.json';
-import finalhopefully from './finalhopefully.json';
-import fixedinterweave from './fixedinterweave.json';
-import final from './FAR_TIME.json';
-import last from './lastone.json';
-import am645 from './645am.json';
-import sunday2 from './sunday2.json';
-import june2 from './6_2.json';
-import june2_2 from './6_2_522am.json';
+import main_data from './main_data.json'
```
Comment (mpkarpov-ui) [inline Wind-Model/my-app/src/index.js:9]: Nice!

File: Wind-Model/programdata/KEY_ROTATE_META.txt
Comment (mpkarpov-ui) [inline Wind-Model/programdata/KEY_ROTATE_META.txt:1]: Let's consider putting this in a gitignore maybe, can be a followup

**Review by mpkarpov-ui — APPROVED** [review]
We should probably archive the json_files directory, this is fine for now but it makes the repo messy. I haven't tested locally so once mihir tests it out LGTM!

File: Wind-Model/programdata/KEY_ROTATE_META.txt
Comment (skb2014) [reply Wind-Model/programdata/KEY_ROTATE_META.txt:1]: Aye aye captain ❤️

**Pull request merged**